CHANGELOG.md

```diff
@@ -3,6 +3,43 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 2024-05-27
+
+### Changes
+
+---
+
+Packages with breaking changes:
+
+ - There are no breaking changes in this release.
+
+Packages with other changes:
+
+ - [`ubuntu_localizations` - `v0.4.0`](#ubuntu_localizations---v040)
+ - [`ubuntu_widgets` - `v0.6.0`](#ubuntu_widgets---v060)
+ - [`ubuntu_lints` - `v0.3.1`](#ubuntu_lints---v031)
+ - [`ubuntu_test` - `v0.1.0-beta.10`](#ubuntu_test---v010-beta10)
+
+Packages with dependency updates only:
+
+> Packages listed below depend on other packages in this workspace that have had changes. Their versions have been incremented to bump the minimum dependency versions of the packages they depend upon in this project.
+
+ - `ubuntu_test` - `v0.1.0-beta.10`
+
+---
+
+#### `ubuntu_localizations` - `v0.4.0`
+
+- Bump Flutter to `3.22.1`
+- 
+#### `ubuntu_widgets` - `v0.6.0`
+
+- Bump Flutter to `3.22.1`
+
+#### `ubuntu_lints` - `v0.3.1`
+
+- Bump ubuntu_lints to `4.0.0`
```

Comment:
I suspect you meant "Bump **flutter_lints** to `4.0.0.`", didn't you?

Reply:
I did indeed, noticed it directly after I published 😄